src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores-export-button.component.ts

```diff
@@ -126,7 +126,7 @@ export class ExerciseScoresExportButtonComponent {
         const { participantName, participantIdentifier } = participation;
         const score = roundValueSpecifiedByCourseSettings(result.score, getCourseFromExercise(exercise));
 
-        const columns = [participantName, participantIdentifier, score, resultWithPoints.totalPoints];
+        const columns = [`"${participantName}"`, participantIdentifier, score, resultWithPoints.totalPoints];
```

Comment:
I am not sure if the name can legally contain `"` characters, but as a precaution maybe it may make sense to escape `"` characters by replacing them with `""` to handle this potential case too.